README.md

```diff
@@ -1,38 +1,73 @@
+
+### Try the Demo
+
+Use the [YubiKit Demo App](./yubikit-android/tree/master/YubikitDemo) to learn how to integrate the YubiKit for Android with your app. The Demo app, which is implemented in Kotlin, shows several examples of how to use YubiKit, including WebAuthn/FIDO2 over the accessory or NFC YubiKeys. It shows how the library is linked with a project so it can be used for a side-by-side comparison when adding the library to your own project.
+
+Open the YubiKitDemo Android Studio project and run it on a real device or an emulator to see the features.
+
+### Integrating and Using the Library
+
+The YubiKit SDK is available as a multi-module library to be added as a dependency to your Android project. The instructions for integrating and using each module are in the README for each of the modules listed above in the About section, which also provides a link to each README.
 
-This library comes with a demo application named **YubikitDemo**. The application is implemented in Kotlin.
 
 ## FAQ <a name="faq"></a>
 
-#### Q1. Are there any versions of Android where YubiKit does not work?
+### Q1. Does the YubiKit work with all versions of Android?
+
+A1. All YubiKit modules should work on Android API 19+. Yubico typically tests and supports n-1 per https://en.wikipedia.org/wiki/Android_version_history.
+
+### Q2. How can I debug my app on Android when a YubiKey takes up the USB port?
+
+A2. You can set up Android Debug Bridge (adb) debugging over WiFi: https://developer.android.com/studio/command-line/adb#wireless
+
+### Q3.  Why is the USB device permissions prompt being shown every time the YubiKey is connected?
 
-All YubiKit modules should work on Android API 19+
-Yubico typically tests and support n-1 per https://en.wikipedia.org/wiki/Android_version_history.
+A3. This is an Android limitation. Android deals with these permissions at the OS level, and the only workaround is to use an intent filter as described in the Android Developers Guide [USB Host Overview](https://developer.android.com/guide/topics/connectivity/usb/host.html#using-intents). However, while this gets rid of the permissions prompt, it also causes your app to launch automatically whenever the YubiKey is connected.
 
-#### Q2. How can I debug my app on Android when a YubiKey takes up the USB port?
 
-You can set up adb debugging over WiFi: https://developer.android.com/studio/command-line/adb#wireless
+## Additional Resources
 
-## Additional resources <a name="additional_resources"></a>
-1. Yubico - [Developers website](https://developers.yubico.com)
-2. Yubico - [Online Demo](https://demo.yubico.com)
-![]()
+* Yubico - [Developers' website](https://developers.yubico.com)
+* Yubico - [Online Demo](https://demo.yubico.com) for OTP and U2F
```

Comment:
Replace U2F with WebAuthn.
The Playground part of the demo supports both YubiOTP and OATH, not sure if we want to leave it at just "OTP" as it is now, or if we want to specify.

Reply:
I specified "for YubiOTP, OATH and WebAuthn"